src/SmartComponents/Inventory/applications/vulnerabilities/DataMapper.js

```diff
@@ -18,13 +10,21 @@ export function createCveListBySystem({ isLoading, ...rest }) {
             data: data.map(row => ({
                 id: row.id,
                 cells: [
-                    <span key={ row.id }>
-                        <SecurityIcon size="md" color={ colorToImpact[row.attributes.impact] } />
-                    </span>,
+                    <Shield
+                        impact={ row.attributes.impact }
+                        hasTooltip={ true }
+                        tooltipPosition={ 'right' }
+                        key={ row.id.toString() }
+                    />,
                     row.attributes.synopsis,
-                    <span key={ `title-${row.id}` } title={ row.attributes.description }>{ row.attributes.description.substr(0, 199) }...</span>,
+                    <LongTextTooltip
+                        content={ row.attributes.description }
+                        tooltipMaxWidth={ '50vw' }
+                        entryDelay="1200"
```

Comment:
Let's keep the delay for tooltips consistent with Patternfly's specs

Reply:
@jiridostal required that. Jirko, please leave a comment.

Reply:
If the entry delay is on default value in this case tooltips get very annoying as there is large hover area in each row of table and the tooltip content is usually very long. Scrolling on the page is very uncomfortable with the default entry delay - so we may need UXD support on this, however keeping the default pretty much ruins the experience.

Reply:
@jiridostal Makes sense to me. Let's make a note to get UXD on this, but it'll be fine for now.